src/tests/core_spec.cljs

```diff
@@ -1,4 +1,4 @@
-(ns status-im.core-spec
```

Comment:
not entirely sure if this should move, perhaps we should just combine all component specs ns' into this file.